JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/query/StoreRouter.java

```diff
@@ -163,6 +197,53 @@ public static StoreClientInterface getStoreClient(String targetResourceID, boole
 		return kbClient;
 	}
 	
+	/**
+	 * Get store client for ontokgrouter
+	 */
+	@Override
+	public StoreClientInterface getRouterStoreClient() {
+		return new RemoteStoreClient(storeRouterEndpoint);
+	}
+	
+	@Override
+	public List<String> getFromStore(String targetResourceLabel, StoreClientInterface storeClient){
```

Comment:
What does it get from the store? I think a comment is required here to let users of this public method know what they will get if they call it.